### DISCUSSION TITLE
Dart in API Docs

### DISCUSSION THREAD

**Discussion by jtkeyva** [supabase/supabase#9888] [Feature Requests]
Seems trivial to add Dart to the API Docs along with JavaScript and Bash. Dart/Flutter is skyrocketing, would save a lot of time for a lot of Dart/Flutter devs over the years and likely a quickish addition to the API Docs.

**Comment by dshukertjr** [comment]
Hey @jtkeyva,
Which docs are you referring to here? Is it [this one](https://supabase.com/docs/reference)?

**Reply by jtkeyva** [reply]
I don't exactly recall the specifics but it was in:
https://supabase.com/docs

Seems to be ok now that I've scanned through. Perhaps a mistake on my end...

**Comment by JediPixels** [comment]
Inside the Supabase dashboard when viewing the Tables, or Auth you click on the <> API Docs button and it slides the sample API calls on the right side of the screen. It only shows Javascript and Bash.

<img width="1916" alt="Screenshot 2024-04-16 at 1 28 12 AM" src="https://github.com/supabase/supabase/assets/43501910/5b27c57f-9d9b-42af-9944-f4ac3a5a7a6c">
<img width="1924" alt="Screenshot 2024-04-16 at 1 26 01 AM" src="https://github.com/supabase/supabase/assets/43501910/b516b90b-06fa-4e76-9479-3b32f91f9516">

It would be amazing to add Dart and possibly TypeScript.

Thanks.

**Comment by JediPixels** [comment]
Any updates on this feature? I sure would love to see it.

Thanks
